pkg/generate/app/cmd/newapp.go

```diff
@@ -451,6 +470,7 @@ func (c *AppConfig) Run(out io.Writer) (*AppResult, error) {
 	}
 
 	objects = app.AddServices(objects)
+	objects = app.AddRoutes(objects)
```

Comment:
Hmmm, I don't remember that. Anyway, how should it be done then? Another flag to enable it?